app/views/dashboard/_user_create.html.erb

```diff
@@ -1,32 +1,34 @@
 <span class="xit-box" id="exit-user-menu">
   <%= image_tag("http://icons.iconarchive.com/icons/oxygen-icons.org/oxygen/24/Actions-application-exit-icon.png") %>
 </span>
-<h3>Create new user below</h3>
-
-  <hr>
-  <div class="form-group">
-    <input class="user-fields" placeholder="https://plus.google.com/num_id" type="text" id="user-profile">
-  </div>
-  <input class="submit-button" id="user-submit" type="button" name="submit" value="Add New Trelora User">
-<br />
-<br />
-<h3>Create new field worker below</h3>
-
-  <hr>
-    <div class="form-group">
-      <%= text_field_tag 'f-w-username', nil, placeholder: "antonio@trelora.com", class: "user-fields" %>
-      <div class="f-w-info-wrapper">
-        <div id="trelora-id-field">
-          <%= text_field_tag "f-w-trelora-id", nil, placeholder: "trelora id" %>
-        </div>
-        <div class="job-role-button">
-          <input type="radio" class="radio_item" value="photographer" name="role" id="photographer">
-          <label class="label_item" for="photographer"><%= (image_tag "camera_pictogram.png", width: "30px" ) %></label>
-        </div>
-        <div class="job-role-button">
-          <input type="radio" class="radio_item" value="appraiser" name="role" id="appraiser">
-          <label class="label_item" for="appraiser"><%= (image_tag "appraiser.jpg", width: "30px" ) %></label>
-        </div>
-      </div>
-      <%= submit_tag "Add a new field worker", class:"submit-button", id:"f-w-submit" %>
+<h3>Create new user</h3>
+<hr>
+<div class="form-group">
+  <%= form_for(User.new, url: "/api/v1/users") do |f| %>
```

Comment:
@chadellison This takes us away from posting using ajax calls.
It causes a call to the api/v1/users create action instead of using ajax to call an api endpoint and return the information. This then redirects to the root page causing the whole page to be refreshed. 
I imagine this was done to get the flash message on the screen. Instead we want to create a custom flash message in JS probably.